main/guides/UIComponentLibrary/UIComponentsstart.md

```diff
@@ -0,0 +1,109 @@
+
+You'll find a collection of ready-to-use or pre-coded user interface elements that help accelerate your Agoric Dapp-building. These pre-designed elements include wallet connect buttons, chain selector, node selector, amount input and wallet provisioning.
+Utilization of these building blocks ensures best practices in design and accessibility.
+
+## Installation and Setup
+
+See [Introduction to Agoric UI Kit](https://a0a31cba.documentation-7tp.pages.dev/guides/getting-started/how-to-use-ui-kit#introduction-to-agoric-ui-kit) for installation and setup how-to.
+
+Once complete you'll need to run this yarn command: 
+
+```sh
+yarn add @agoric/react-components
+```
+
+## UI Components List
+
+Before getting started, there are some resources you might want to keep handy in case you get stuck, have questions, or are curious about any of the components. Getting in contact with us is easy!
+
+- Connect Wallet
+- Wallet Provisioning
+- Amount Input
+- Chain Selector
+- Node Selector
+
+## Connect Wallet
+
+<img style="border: 2px solid grey"
+  alt="Screenshot: Connect Wallet"
+  src="./assets/connectwallet.png" />
```

Comment:
> RollupError: Could not resolve "./assets/connectwallet.png" from "main/guides/UIComponentLibrary/UIComponentsstart.md"

The `assets` directory is one level up, but does not contain a `connectwallet.png` file. Maybe it should be this?
```suggestion
  src="../assets/5-ConnectWalletUI.png" />
```
(and likewise for the other \<img>s)